### ISSUE TITLE
Unable to track the kinect

### ISSUE THREAD

**Issue by ColletAxel** [lvsn/6DOF_tracking_evaluation#3]
Hello,
I've successfully train the dragon tracker but I currently try to do the same things with the kinect dataset.
The tracker leave the object's localisation everytime (the translation and rotation localisations). 
I Adapt the object's localisation format of the dataset 2 for be equivalent to the dataset 1. If I set the "reset_frequency" parameter in the json file, I can see a good repositioning.
In the real data generation, I've supress the saturation_threshold because the Image generate was very impact by this feature.
For all over parameters I use same values than the dragon and 

I've try all the neural network architecture in '.lua' provide in 'deeptracking/tracker'

Did you achieve to track the kinect with the 1st version of the deeptracking and which architecture did you use

Thanks !

Axel Collet
![out](https://user-images.githubusercontent.com/29093782/49290618-bbaa0e00-f4a7-11e8-8b2c-e887aff5b327.gif)

**Comment by MathGaron** [comment]
Hey Axel,
The tracking seems totally off. Removing the saturation threshold is a good thing because of the color of the kinect.
Could you show me an example of real and synthetic images that you are using. One thing could be the bounding box used around the object. (However a too small bounding box on this object would just make the tracking slide on the "long" axis. but a too large bounding box could be the problem).
Also, do you train your network with the synthetic and the real at the same time or you do a training and then finetune with real data? training with both at the same time will help the network.
I will check this out on my side tomorrow.

**Comment by MathGaron** [comment]
Hey Axel, were you able to fix the problem?

**Comment by ColletAxel** [comment]
Hi, 
I've try with only synthetic data and a merge of real and synthetic and I've no solution for track the Kinect. I've switch on over model who work well.

**Comment by MathGaron** [comment]
Have you tried with our latest architecture (with only synthetic)? Is it the only model who fails to work or you had problem with others in the dataset?

Thank you for the reply

**Comment by ColletAxel** [comment]
No, I've try only with the first one. I've switch to other model who works well. so I gave up to track the white kinect model. In the training I've observe the badest val loss compare to other model. In exemple, here is one of the best result in function of training loss and validation loss with only synthetic data.

![loss](https://user-images.githubusercontent.com/29093782/52212700-74684100-288d-11e9-938f-9b75cdfff273.png)

we see well than the validation loss is never under 0.10. in the other model training i always have 0.5 in the best result. I have find no explication. Maybe just it's a symmetric object with less visual characteristic.


Thank you for your attention to my problem.

**Comment by MathGaron** [comment]
Hey,

Indeed I never trained on the kinect model with the old architecture, however I have seen that kind of behavior you see after epoch 16, usually dividing the learning rate by 10 before it happens might help.

My guess for the bad loss is that the kinect is "longer" which makes most of the input "black". Do you have the curves for each degree of freedom? Might help to see which degree of freedom fails.
